### DISCUSSION TITLE
is_in_transport_position as FieldDeviceCoreDetails

### DISCUSSION THREAD

**Discussion by AdamMnDOT** [usdot-jpo-ode/jpo-wzdx#367] [Ideas]
It seems like the "is_in_transport_position" field is actually relevant to more than just arrow boards and is equally relevant to Dynamic Message Signs in particular.  I believe the "is_in_transport_position" field could have value to all the device types and allow filtering of devices being setup but not displayed to traffic yet or other similar situations.  I propose having "is_in_transport_position" as a "FieldDeviceCoreDetails" field instead of just specifically for Arrow Boards.

**Comment by mark-mockett** [comment]
thanks @AdamMnDOT - the WZDx spec here isn't being updated anymore, but the content is currently being adapted into a [standard](https://www.ite.org/technical-resources/standards/cwz/) by ITE and SAE. We'll discuss this idea with that group.
